src/Destructurama.ByIgnoring/Destructurama.ByIgnoring.csproj

```diff
@@ -2,9 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.1;netstandard2.0;netstandard1.0</TargetFrameworks>
-    <LangVersion>latest</LangVersion>
     <VersionPrefix>2.0.1</VersionPrefix>
```

Comment:
Addition of NRT annotations probably qualifies as a feature increment - could move to 2.1.0 on `dev` now?

Reply:
In fact dev and master branches have changed a role of each other. master is for non-breaking changes. dev is for breaking changes.